components/ai_chat/resources/page/components/filter_menu/tools_menu.test.tsx

```diff
@@ -7,13 +7,42 @@ import '$test-utils/disable_custom_elements'
 
 import * as React from 'react'
 
-import { render } from '@testing-library/react'
+import { render, screen, fireEvent } from '@testing-library/react'
 import { ActionType } from 'gen/brave/components/ai_chat/core/common/mojom/ai_chat.mojom.m.js'
 import ToolsMenu from './tools_menu'
+import { AIChatReactContext } from '../../state/ai_chat_context'
+import * as Mojom from '../../../common/mojom'
+
+// Mock locale function
+jest.mock('$web-common/locale', () => ({
+  getLocale: (key: string) => {
+    const mockStrings: { [key: string]: string } = {
+      CHAT_UI_SMART_MODES_GROUP: 'Smart Modes',
+    }
+    return mockStrings[key] || key
+  },
+}))
```

Comment:
hey, this should automatically be mocked in tests. Additionally, rather than adding a translation here, we should just use the key `CHAT_UI_SMART_MODES_GROUP` in the tests